react/Gallery.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import GalleryLayout, { GalleryLayoutPropsWithSlots } from './GalleryLayout'
+//@ts-ignore
```

Comment:
Is this because of `.js` extension in `GalleryLegacy`?
Have you tried adding `allowJs` in the `tsconfig` file?

Reply:
Yes, it is. Did not try that, thank you!

Reply:
Builder-hub overrides this config 😓. There's another alternative to that?